inc/breadcrumbs.php

```diff
@@ -182,6 +197,9 @@ function nightingale_breadcrumb_trail() {
 			$home_page = get_option( 'page_on_front' );
 			foreach ( $ancestors as $ancestor ) {
 				if ( ( end( $ancestors ) !== $ancestor ) && ( ( $home_page !== $ancestor ) ) ) {
+					if ( (int) $home_page === $ancestor ) {
```

Comment:
would this be better combined into the line above:
`&& ( ( ( int ) $home_page !== $ancestor ) ) )`

Possibly cleaner...?